### PR TITLE
fix(docs): generate spec.md

### PR DESCRIPTION
**Pull request by aesteves60**
Script for generate spec.md for each component

- Edit script `generate-typedoc-md.js` now not for all component but for one specifique
- Move script `generate-typedoc-md.js` to `packages/common/core/script` => `scripts`
- In `common/core` script `yarn doc` remove execution of `generate-typedoc-md.js` & transpilation of `typedoc-json-to-md`
- Expose `typedoc-json-to-md` in `@ovhcloud/ods-common-core`

- You can pass a multiple arg to the script for the component with multiple component like Accordion or Radio

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1509.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1509.1 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#1514.0 ✘  
* 
  * build ✘

File: packages/components/accordion/package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "ignore:rm": "git clean -Xdf",
     "generate": "stencil generate",
     "doc:api": "typedoc",
+    "doc": "yarn run doc:accordion && yarn run doc:accordion-group && node ../../../scripts/generate-typedoc-md.js --multiple",
+    "doc:accordion": "typedoc src/components/osds-accordion/public-api.ts --out ./docs-api/accordion --json ./docs-api/accordion/typedoc.json",
+    "doc:accordion-group": "typedoc src/components/osds-accordion-group/public-api.ts --out ./docs-api/accordion-group --json ./docs-api/accordion-group/typedoc.json",
```
Comment (dpellier) [inline packages/components/accordion/package.json:27]: Food for thought, maybe those (and radio) should be separated into different components

**Review by dpellier — APPROVED** [review]

**Pull request merged**